lib/ex_unit/lib/ex_unit/diff.ex

```diff
@@ -66,6 +71,18 @@ defmodule ExUnit.Diff do
 
   def format(_left, _right, _formatter), do: nil
 
+  defp proceed_string_difference?(left, right) do
+    sum_length = String.length(left) + String.length(right)
```

Comment:
Since the algorithm complexity is calculated by multiplying both lengths, should we calculate this function using multiplication? I am not sure, it is up to you, other than that, :shipit: :)